docs/source/tips.rst

```diff
@@ -46,3 +46,62 @@ Here is an instruction to install NumPy with OpenBLAS using `Homebrew <https://b
 If you want to install NumPy with pip, use `site.cfg <https://github.com/numpy/numpy/blob/master/site.cfg.example>`_ file.
 
 For details of this problem, see `issue #704 <https://github.com/chainer/chainer/issues/704>`_.
+
+
+How do I accelerate my model using iDeep on Intel CPU?
+------------------------------------------------------
+
+Follow these steps to utilize iDeep in your model.
+
+Install iDeep
+~~~~~~~~~~~~~
+
+The following environments are recommended by `iDeep <https://github.com/intel/ideep>`_.
+
+* Ubuntu 14.04 / 16.04 LTS (64-bit) and CentOS 7 (64-bit)
+* Python 2.7.5+, 3.5.2+, and 3.6.0+
+
+On recommended systems, you can install iDeep wheel (binary distribution) by:
+
+.. code-block:: console
+
+    $ pip install ideep
+
+Enable iDeep Configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+iDeep is disabled by default.
```

Comment:
iDeep is disabled by default, because it's an experimental feature (in my understanding).
How about mentioning that?

Reply:
I agree. Fixed to mention that fact.